src/main/scala/com/ironcorelabs/SucceededResult.scala

```diff
@@ -0,0 +1,6 @@
+package com.ironcorelabs.scala.sdk
+
+private[sdk] object SucceededResult {
+  def apply(fromJava: com.ironcorelabs.sdk.SucceededResult): List[UserOrGroupId] =
```

Comment:
quick gut check: Is it idiomatic to have an `apply` not return a List of another type? It took me a minute to figure out what was going on here.

An alternative might be to make this an actual wrapper that also defined a `toList` or similar. It would make the call site more easily understandable, but maybe has other problems.

Reply:
Moved.